services/routes/civic.js

```diff
@@ -1,12 +1,11 @@
 const https = require('https');
 const axios = require('axios');
 const apiKey = process.env.NODE_GOOGLE_CIVIC_API_KEY;
```

Comment:
Environment variable  that can be updated for deployment. For 2020 GA senate runoffs it will be set to 7001 which is the Civic APIs id for this election.